packages/components/input/src/input.ts

```diff
@@ -166,6 +166,14 @@ export const inputProps = buildProps({
     type: Number,
     default: 2,
   },
+  borderless: {
+    type: Boolean,
+    default: false,
+  },
```

Comment:
```suggestion
  borderless: Boolean,
```
And please add a comment above.